### DISCUSSION TITLE
Feature Request: Implement Vimium-like Keyboard Navigation for Meta Quest 3 Browser

### DISCUSSION THREAD

**Discussion by RioFuji** [philc/vimium#4535] [General]
Dear Vimium Team,

I hope this message finds you well. I am writing to request the implementation of Vimium-like keyboard navigation for the Meta Quest 3 browser.

As a user of the Meta Quest 3, I have found that browsing using traditional pointing devices, hand tracking, and virtual keyboards can be cumbersome and less efficient compared to keyboard-based navigation. Given the unique constraints and user experience challenges associated with VR browsing, having a powerful and intuitive keyboard navigation tool like Vimium would greatly enhance usability and efficiency.

Vimium’s keyboard shortcuts for navigating and controlling web pages provide a seamless and highly productive browsing experience on desktop browsers. Implementing similar functionality in the Meta Quest 3 browser would address the current difficulties users face with the available input methods, making browsing in VR much more fluid and manageable.

I believe that incorporating Vimium-like features would not only improve the overall browsing experience for Meta Quest users but also attract a broader audience who prefers keyboard-based navigation. Your consideration of this request would be greatly appreciated.

Thank you for your time and for your continued efforts in enhancing web navigation tools.

Best regards

**Comment by rio675** [comment]
I could install the vimium feature with the combination of mobile vr station and kiwi browser apk.
